From 003f19030687474afc1c4633e448b520ecf88da3 Mon Sep 17 00:00:00 2001
From: Daniel Leung <daniel.leung@intel.com>
Date: Wed, 13 Mar 2019 16:18:08 -0700
Subject: [PATCH 1/2] xtensa: enable remainderf

Xtensa libgcc does not supply this, so need to enable in libm.

Signed-off-by: Daniel Leung <daniel.leung@intel.com>
---
 newlib/libm/math/ef_remainder.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/newlib/libm/math/ef_remainder.c b/newlib/libm/math/ef_remainder.c
index 5371009f..4c805f90 100644
--- a/newlib/libm/math/ef_remainder.c
+++ b/newlib/libm/math/ef_remainder.c
@@ -17,8 +17,6 @@
 
 /* __ieee754_remainderf is provided by libgcc */
 
-#if 0
-
 #ifdef __STDC__
 static const float zero = 0.0;
 #else
@@ -70,5 +68,3 @@ static float zero = 0.0;
 	SET_FLOAT_WORD(x,hx^sx);
 	return x;
 }
-
-#endif /* 0 */
-- 
2.19.2

